proftpd/init.sls

```diff
@@ -17,6 +17,7 @@ proftpd:
         - mode: 644
         - template: jinja
         - defaults:
+            modules_config: {{ salt['pillar.get']('proftpd:modules_config') }}
```

Comment:
those changes are affecting the formula behaviour not only kitchen so I think they should be separated in their own commit with the correct naming (feat: maybe?)

Reply:
> those changes are affecting the formula behaviour not only kitchen so

what do you mean?

Reply:
I mean that in your commit there are changes to kitchen/inspec like indicated in the title `ci(kitchen): add Kitchen tests` but there also are changes which are more 'invasive' because they change the formula code not only the CI/tests. I think those changes should be in a separated commit named `feat` or `refactor`

Reply:
yes... maybe. but I don't have more time to spend on this PR...

Reply:
@n-rodriguez No problem, I'll try to get around to this soon to finalise and merge.